draft-ietf-dnsop-svcb-https.md

```diff
@@ -114,12 +114,6 @@ and introductory examples, much of this document refers only to the SVCB RR,
 but those references should be taken to apply to SVCB, HTTPS,
 and any future SVCB-compatible RR types.
 
-The SVCB RR has two forms: 1) the "Alias Form" simply delegates operational
-control for a resource; 2) the "Service Form" binds together
-configuration information for a service endpoint.
-The Service Form provides additional key=value parameters
-within each RDATA set.
-
```

Comment:
While repetitive with what is below, this does provide important background context.  Should we see if we can include this even if we condense this down some?  Or just leave this as-is?